test/test.ts

```diff
@@ -1,5 +1,26 @@
-describe('this package', () => {
-  it('should have tests', () => {
-    // Write your test
-  });
+import { getRelHref } from '../src/user/service';
```

Comment:
Move this file to `test/user/service.ts`

(note that there's already other tests in this pacakge)